script/Arcueid/Day 10/10_00_ARC10_5B.txt

```diff
@@ -442,7 +446,7 @@ I just thought that if we created normal memories together, then she would defin
 [sha:5bfe9ae1bed5825c25a954ae51f570bd8133b5ac]{
 -- Page 498, Offset 25556.
 -- 「……まあ、おまえが忙しいっていうんなら別にいいけど」
-"...Well, if you don't have the time for that it's okay."
+"...Well, if you don't have the time for that, it's okay."
```

Comment:
Might revert unless he actually pauses in the va

Reply:
I think this pause is pretty necessary to read it in English. Reading it aloud with and without it sounds kinda awkward and run-on-y without the comma.